frontend/src/app/main/ui.cljs

```diff
@@ -72,7 +72,15 @@
         :dashboard-team-settings)
        [:*
         #_[:div.modal-wrapper
-           [:& app.main.ui.onboarding/release-notes-modal {:version "1.8"}]]
+           ;; modal de la derecha
```

Comment:
In spanish? :smile:

Reply:
yeah, that was a quick and dirty comments for the @elhombretecla  to know which component to touch :P